ruleset.go

```diff
@@ -132,37 +133,24 @@ type RuleActionExtraction struct {
 	Regex  string `json:"regex,omitempty"`
 }
 
-// ListRulesets gets all rulesets.
-func (c *Client) ListRulesets() (*ListRulesetsResponse, error) {
-	rulesetResponse := new(ListRulesetsResponse)
-	rulesets := make([]*Ruleset, 0)
-
-	// Create a handler closure capable of parsing data from the rulesets endpoint
-	// and appending resultant rulesets to the return slice.
-	responseHandler := func(response *http.Response) (APIListObject, error) {
-		var result ListRulesetsResponse
-		if err := c.decodeJSON(response, &result); err != nil {
-			return APIListObject{}, err
-		}
-
-		rulesets = append(rulesets, result.Rulesets...)
-
-		// Return stats on the current page. Caller can use this information to
-		// adjust for requesting additional pages.
-		return APIListObject{
-			More:   result.More,
-			Offset: result.Offset,
-			Limit:  result.Limit,
-		}, nil
+// ListRulesetsWithOpts get all rulesets with options
+func (c *Client) ListRulesetsWithOpts(o APIListObject) (*ListRulesetsResponse, error) {
+	v, err := query.Values(o)
+	if err != nil {
+		return nil, err
 	}
-
-	// Make call to get all pages associated with the base endpoint.
-	if err := c.pagedGet("/rulesets/", responseHandler); err != nil {
+	resp, err := c.get("/rulesets?" + v.Encode())
```

Comment:
Because this removes the pagination it would be a breaking API change, can you revert back to using `pagedGet()`?